create-view.js

```diff
@@ -65,9 +65,6 @@ module.exports = function (opts) {
           if (!temporary) {
             sourceDB._cachedViews = sourceDB._cachedViews || {};
             sourceDB._cachedViews[viewSignature] = view;
-            view.db.on('destroyed', function () {
-              delete sourceDB._cachedViews[viewSignature];
-            });
```

Comment:
To be sure, it worries me that this code wasn't getting hit anymore. But 1) the tests still pass, and 2) map/reduce will be folded into pouchdb core soon enough when I add native indexes. So I'm not too worried.